FirebaseAdmin/FirebaseAdmin/Auth/AuthErrorCode.cs

```diff
@@ -29,6 +29,13 @@ public enum AuthErrorCode
         /// </summary>
         EmailAlreadyExists,
 
+        /// <summary>
```

Comment:
Add to the end of this enum. C# assigns an integer to each enum value depending on their position, and also lets developers access that integer value:

```cs
var code = AuthErrorCode.UserNotFound;
Console.WriteLine((int)code);
```

It is a breaking change to modify the int values (kind of like the field position in protos)

Reply:
Thanks for catching and explaining. I moved it to the end of the enum.